src/legacy/core_plugins/vis_type_vega/public/vega_view/vega_tooltip.js

```diff
@@ -68,7 +68,7 @@ export class TooltipHandler {
     // Sanitized HTML is created by the tooltip library,
     // with a large number of tests, hence suppressing eslint here.
     // eslint-disable-next-line no-unsanitized/property
-    el.innerHTML = createTooltipContent(value, _.escape);
+    el.innerHTML = createTooltipContent(value, _.escape, 2);
```

Comment:
What is this additional parameter `2` here for?

Reply:
@majagrubic that's `maxDepth` -- https://github.com/vega/vega-tooltip/blob/93644790c2db20b9ff7aa8330484d632b252642e/src/formatValue.ts#L9

Reply:
Right, this is how many levels the stringifier goes into the object to create a string representation. I figured 2 is enough here because the space in the tooltip is really limited.
<img width="425" alt="Screenshot 2019-10-10 at 11 29 39" src="https://user-images.githubusercontent.com/1508364/66556877-45f3e180-eb51-11e9-9b4c-7ec8001ca2ff.png">

If we change how the tooltip renders the data (e.g. if we add line wrapping), we can re-visit this